src/views/ProfileView.vue

```diff
@@ -0,0 +1,322 @@
+<script setup lang="ts">
+  import LayoutCard from '@/components/cards/LayoutCard.vue';
+  import { ref, type Ref, onBeforeMount } from 'vue';
+  import { useI18n } from 'vue-i18n';
+  const { t }: { t: Function } = useI18n();
+  type LabelValue = {
+    label: string;
+    labelAbbr?: string;
+    value: string;
+  };
+
+  import { usePersonInfoStore, type PersonInfoStore, type PersonInfoResponse } from '@/stores/PersonInfoStore';
+  import { usePersonenkontextStore, type Zuordnung, type PersonenkontextStore } from '@/stores/PersonenkontextStore';
+  import { OrganisationsTyp } from '@/stores/OrganisationStore';
+
+  export type SchoolData = {
+    title: string;
+    info?: string | null;
+    schoolAdmins?: string[];
+    labelAndValues: LabelValue[];
+  };
+
+  const personInfoStore: PersonInfoStore = usePersonInfoStore();
+  const personenkontextStore: PersonenkontextStore = usePersonenkontextStore();
+  const personalData: Ref = ref<LabelValue[]>([]);
+  const schoolDatas: Ref = ref<SchoolData[]>([]);
+
+  function handleGoToPreviousPage(): void {
+    window.history.back();
+  }
+
+  function groupZuordnungen(zuordnungen: Zuordnung[]): Map<string, Zuordnung[]> {
+    const groupedZuordnungen: Map<string, Zuordnung[]> = new Map();
+    for (const zuordnung of zuordnungen) {
+      if (groupedZuordnungen.has(zuordnung.sskDstNr)) {
+        groupedZuordnungen.get(zuordnung.sskDstNr)?.push(zuordnung);
+      } else {
+        groupedZuordnungen.set(zuordnung.sskDstNr, [zuordnung]);
+      }
+    }
+    return groupedZuordnungen;
+  }
+
+  function createComposedZuordnungen(groupedZuordnungen: Map<string, Zuordnung[]>): Zuordnung[] {
+    const composedZuordnungen: Zuordnung[] = [];
+    for (const [, zuordnungen] of groupedZuordnungen) {
+      if (zuordnungen.length > 1) {
+        const aggregatedRoles: string[] = zuordnungen.map((z: Zuordnung) => z.rolle);
+        const klasse: string | null =
+          zuordnungen.filter((z: Zuordnung) => z.typ === OrganisationsTyp.Klasse).at(0)?.sskName ?? null;
```

Comment:
Wir können den Fall haben, wo eine Person in der gleichen Schule in mehreren Klassen ist. Hier aber nimmst du nur die erste Klasse.

Reply:
Das war mir nicht bewusst. Habe es nun abgeändert. Die verschiedenen Klassen werden nun kommagetrennt in der Zeile Klasse aufgeführt